.gitignore

```diff
@@ -5,3 +5,4 @@ doc
 benchmark/implementations/*
 !/benchmark/implementations/current/
 .idea
+package-lock.json
```

Comment:
The package lock should either be committed or ignored. Seeing as it isn't committed, I'm ignoring it.

Reply:
Don't touch configs if PR is not related